### PR TITLE
 [CPM-68] Creating Home Page

### PR DESCRIPTION
**Pull request by FocalChord**

### CODE REVIEW

File: client/src/pages/HomePage/index.js [hunk 1]
```diff
@@ -1,23 +1,49 @@
-import React, { useState } from "react";
-import { Button } from "../../components";
-import { MomentumClient } from "../../common";
+import React from "react";
+import { Flex, Text, Button, Image } from "@chakra-ui/core";
+import { Link } from "react-router-dom";
+
+const HomePageButton = ({ text, to }) => {
```
Comment (vee-christella) [inline client/src/pages/HomePage/index.js:5]: Just to explicitly clarify, are we using functional components or classes? We should keep it consistent throughout the app. Happy with either 😄

**Review by vee-christella — APPROVED** [review]
LGTM 👍

File: client/src/pages/HomePage/index.js (hunk 1 above)
Comment (Dinith1) [reply client/src/pages/HomePage/index.js:5]: Let's use functional components please 😺

Comment (Dinith1) [reply client/src/pages/HomePage/index.js:5]: (this is using functional components if that's what you were asking)

Comment (vee-christella) [reply client/src/pages/HomePage/index.js:5]: Yepyep, I just wanted to confirm that it was intentional so we don't have to refactor later!

**Review by Dinith1 — APPROVED** [review]
Noice LGTM 🚀

**Pull request merged**